kubectl-rook-ceph.sh

```diff
@@ -423,15 +630,16 @@ function run_start_debug() {
   [[ -z "${REMAINING_ARGS[0]:-""}" ]] && fail_error "Missing mon or osd deployment name"
   deployment_name="${REMAINING_ARGS[0]}"              # get deployment name
   REMAINING_ARGS=("${REMAINING_ARGS[@]:1}")           # remove deploy name from remaining args
+  set +u
```

Comment:
Does plus +u is set as it turns off the `unset variables as an error` 
SO to avoid failing

Reply:
Correct. I wonder if this was only necessary on mac. I was hitting the same error for the main `debug start` command even though it was working in the CI.

Reply:
Ohkay got it